stock_chart/README.md

```diff
@@ -0,0 +1,21 @@
+# Simple Stock Price Chart #
+
+# Description and Use #
+
+This ![widget](widget.erb) uses an ![endpoint](endpoint.erb) to fetch
+and process stock data.  This data is then graphed using HighCharts,
+showing a simple one-value plot over time.
+
+[Screen Shot](screenshot.png)
+
+This widget uses the ![Quandl](https://www.quandl.com/) api to provide
```

Comment:
api => API (fix capitalization). Please do a global search on this, it appears at least twice.